build.sh

```diff
@@ -88,13 +88,15 @@ build-couch() {
     docker run \
         --mount type=bind,src=${SCRIPTPATH},dst=/home/jenkins/couchdb-pkg \
         -u 0 -w /home/jenkins/couchdb-pkg \
+        --platform linux/amd64 \
```

Comment:
Omitting this seems to cause Docker to pick up whatever image it might have in the cache and run it, even if it's not the native architecture.